py/src/braintrust/wrappers/test_openai.py

```diff
@@ -122,6 +123,46 @@ def test_openai_responses_metrics(memory_logger):
     assert span["metadata"]["provider"] == "openai"
     assert TEST_PROMPT in str(span["input"])
 
+    # Test responses.parse method
+    class NumberAnswer(BaseModel):
+        value: int
+        reasoning: str
+
+    # First test with unwrapped client - should work but no spans
+    parse_response = unwrapped_client.responses.parse(model=TEST_MODEL, input=TEST_PROMPT, text_format=NumberAnswer)
+    assert parse_response
+    # Access the structured output via text_format
+    assert parse_response.output_parsed
+    assert parse_response.output_parsed.value == 24
+    assert parse_response.output_parsed.reasoning
+
+    # No spans should be generated with unwrapped client
+    assert not memory_logger.pop()
+
+    # Now test with wrapped client - should generate spans
+    start = time.time()
+    parse_response = client.responses.parse(model=TEST_MODEL, input=TEST_PROMPT, text_format=NumberAnswer)
+    end = time.time()
+
+    assert parse_response
+    # Access the structured output via text_format
+    assert parse_response.output_parsed
+    assert parse_response.output_parsed.value == 24
+    assert parse_response.output_parsed.reasoning
+
+    # Verify spans are generated
+    spans = memory_logger.pop()
+    assert len(spans) == 1
+    span = spans[0]
+    assert span
+    metrics = span["metrics"]
+    assert_metrics_are_valid(metrics, start, end)
+    assert 0 <= metrics.get("prompt_cached_tokens", 0)
+    assert 0 <= metrics.get("completion_reasoning_tokens", 0)
+    assert span["metadata"]["model"] == TEST_MODEL
+    assert span["metadata"]["provider"] == "openai"
+    assert TEST_PROMPT in str(span["input"])
```

Comment:
should we be tracking an output?

Reply:
good call ill add for responses.create too